doc/org.eclipse.python4capella.doc/doc/developer/DeveloperManual.textile

```diff
@@ -9,3 +9,4 @@ h2. Advanced Python users
 h2. Python4Capella contributors
 
 * "*Create a Python interpreter distribution for Windows*":python/CreateWindowsPythonFile.html Describes how you can recreate the Python interpreter zip file for Windows.
+* "*Generate Python tests*":python/GeneratePythonTests.html Describes how you can Python tests for the Python Capella light metamodel.
```

Comment:
A verb is missing "how you can write/add..."

Reply:
done